src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+import core.basesyntax.objects.Ball;
+import java.util.Random;
+
+public class Lottery {
+    private final Random random;
+    private final ColorSupplier colorSupplier;
+
+    public Lottery() {
+        // SHOULD I MAKE RANDOM AS CLASS FIELD OR JUST AS SIMPLE LOCAL FUNCTION VARIABLE ?
+        this.random = new Random();
+        colorSupplier = new ColorSupplier();
+    }
+
+    public Ball getRandomBall(int maxNum) {
```

Comment:
you dont need any parameters for this method
```suggestion
    public Ball getRandomBall() {
```
![image](https://github.com/mate-academy/jv-lottery/assets/73041091/69f7120e-d3a3-4258-a670-297c381f1725)